bee-network/examples/common/logger.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use env_logger::fmt::Color;
+use log::{debug, error, info, log_enabled, trace, warn};
+
+use std::io::Write;
+
+pub fn init(level_filter: log::LevelFilter) {
```

Comment:
Maybe this filter would be helpful for other crates too, should this be moved out?

Reply:
I don't know what you mean, Sam. The `logger` impl in `bee-common` let's you configure the log level and even allows you to define target filters. So I am actually thinking about removing this logger module entirely.

Reply:
Yes, I guess this would be nice.

Reply:
Not sure what was the conclusion of this, but we already have this piece of code in bee, I would just use it, otherwise we end up rewriting the same stuff for all crate examples.